public/app/AppWrapper.tsx

```diff
@@ -112,13 +111,13 @@ export class AppWrapper extends React.Component<AppWrapperProps, AppWrapperState
                         id="ngRoot"
                         ref={this.container}
                         dangerouslySetInnerHTML={{
-                          __html: appSeed,
+                          __html: '<grafana-app ng-cloak></app-notifications-list></grafana-app>',
```

Comment:
How do you feel about making this a separate component, like `<AngularRoot />` where all the angular stuff can be more contained within there?

Reply:
yea, that's a good idea